packages/xo-vmdk-to-vhd/src/vmdk-generate.js

```diff
@@ -172,9 +172,13 @@ ddb.geometry.cylinders = "${geometry.cylinders}"
       throw new Error('vmdk is bigger than precalculed size ')
     }
     const size = 1024 * 1024
-    while (remaining > 0) {
-      const yieldSize = Math.min(size, remaining)
-      remaining -= yieldSize
+    const fullBuffer = Buffer.alloc(size, 0)
+    while (remaining > size) {
+      yield track(fullBuffer)
+      remaining -= size
+    }
+    const yieldSize = Math.min(size, remaining)
+    if (yieldSize > 0) {
       yield track(Buffer.alloc(yieldSize))
     }
```

Comment:
I believe `Math.min` and `if (yieldSize > 0)` are unnecessary and the code can be simplified to:

```suggestion
      yield track(Buffer.alloc(remaining))
```